mix.exs

```diff
@@ -81,14 +81,15 @@ defmodule Screens.MixProject do
       {:stream_data, "~> 1.1", only: :test},
       {:memcachex, "~> 0.5.5"},
       {:aja, "~> 0.6.2"},
-      {:telemetry, "~> 1.2"},
+      {:telemetry, "~> 1.2", override: true},
```

Comment:
What is this overriding? Maybe we should have a comment explaining it.

Reply:
🫡 https://github.com/mbta/screens/pull/2092/commits/eeaf975d956ddf7bb67f4d7f848cbd384bfb5ca0